src/services/appointments.js

```diff
@@ -0,0 +1,9 @@
+const Appointment = require('../models/appointments');
+
+const getByDateService = async (date) => {
+	const appointments = await Appointment.find({ date });
```

Comment:
Lets discuss what logic should be in `services`. In my view finding data in DataBase is not service logic

Reply:
Absolutely disagree, our service is made unique that we've got appointments and give them by request.
So, it's business-logic and places for it in services.
Best practice ([resource](https://www.codementor.io/@evanbechtol/node-service-oriented-architecture-12vjt9zs9i)):
![image](https://user-images.githubusercontent.com/63366123/189888701-9c1aeb92-c04d-44fe-86a8-79f5be1971f7.png)